C/c4CAPI.cc

```diff
@@ -1059,6 +1060,14 @@ C4IndexUpdater* C4NULLABLE c4index_beginUpdate(C4Index* index, size_t limit, C4E
     return tryCatch<C4IndexUpdater*>(outError, [&] { return index->beginUpdate(limit).detach(); });
 }
 
+C4SliceResult c4index_getName(C4Index* index) noexcept {
```

Comment:
should handle the exception

Reply:
What exception?

Reply:
noexcept means the app will be terminated should exception popped to here. We do something like, tryCatch<C4 ...>

Reply:
Nevermind 😅